solidity/contracts/converter/interfaces/IBancorConverter.sol

```diff
@@ -7,6 +7,8 @@ import '../../utility/interfaces/IWhitelist.sol';
     Bancor Converter interface
 */
 contract IBancorConverter {
+    function converterType() public pure returns (uint8);
```

Comment:
@yudilevi : This is not needed if not actually used on-chain (and it can be added when needed, without breaking any on-chain or off-chain interface).

Reply:
removed